src/Entity/IdeasWorkshop/Theme.php

```diff
@@ -86,11 +86,23 @@ class Theme implements EnabledInterface
      */
     protected $imageName;
 
-    public function __construct(string $name = '', string $imageName = null, bool $enabled = false)
+    /**
+     * @ORM\Column(type="smallint", options={"unsigned": true})
+     *
+     * @Assert\GreaterThanOrEqual(1)
+     *
+     * @Gedmo\SortablePosition
+     *
+     * @SymfonySerializer\Groups("theme_read")
```

Comment:
do we really need to expose this value? 
Using it to sort the Theme collection is not enough?

Reply:
I'll check with front dev, I guess you are right about this, will probably not be needed